docs/lua-records/functions.rst

```diff
@@ -317,6 +317,17 @@ Reverse DNS functions
     $ dig +short AAAA 2001-a-b--1.static6.example.com @ns1.example.com
     2001:a:b::1
 
+.. function:: filterForward(address, masks)
+
+  Used for limiting the output of :func:`createForward` and :func:`createForward6` to a set of netmasks.
```

Comment:
are the addresses in the `masks` list allowed or not allowed?

Reply:
I updated the param text to `  :param masks: A NetmaskGroup; any address not matching the NMG will be replaced by the fallback address.`